lib/galaxy/visualization/plugins/interactive_environments.py

```diff
@@ -357,7 +356,7 @@ def _get_command_inject_env(self):
                 envsets.append(item[2:])
             elif item.startswith('--env'):
                 envsets.append(item[5:])
-        return dict(map(lambda s: string.split(s, '=', 1), envsets))
+        return dict(map(lambda s: s.split('=', 1), envsets))
```

Comment:
If you prefer a dict comprehension:

```suggestion
        return dict(_.split('=', 1) for _ in envsets)
```